jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/udp/tim/TimReceiver.java

```diff
@@ -94,7 +94,7 @@ public void run() {
                // Submit JSON to the OdeRawEncodedMessageJson Kafka Topic
                logger.debug("Publishing JSON TIM...");
 
-               this.timPublisher.publish(jsonObject.toString(), this.timPublisher.getOdeProperties().getKafkaTopicOdeRawEncodedMessageJson());
+               this.timPublisher.publish(jsonObject.toString(), this.timPublisher.getOdeProperties().getKafkaTopicOdeRawEncodedTIMJson());
```

Comment:
@dan-du-car Shouldn't there be a SpatReceiver now as well?

Reply:
We have not yet implemented the spat udp receiver.  Since v2xhub directly sends data to ODE via kafka topic, these receivers are not used by v2xhub.